eclipsePlugin/feature_p2.inf

```diff
@@ -0,0 +1,7 @@
+# Consider this feature to be an update of both earlier versions of SpotBugs
+# and all versions of FindBugs up to 3.0.2, the last release prior to the fork.
+update.matchExp = providedCapabilities.exists(iu | iu.namespace == 'org.eclipse.equinox.p2.iu' && \
+ ((iu.name == '@FEATURE_ID@.feature.group' && iu.version ~= range('[0, $version$)')) || \
+  (iu.name == 'edu.umd.cs.findbugs.plugin.eclipse.feature.group' && iu.version ~= range('[0, 3.0.3)'))))
+update.severity = 0
+update.description = Update FindBugs to SpotBugs.
```

Comment:
Updating SpotBugs to SpotBugs also shows this message, which is a bit awkward.
I would propose to use here same as in our regular feature description:
"SpotBugs is a program which uses static analysis to look for bugs in Java code, a spiritual successor of FindBugs."